docs/4.x/tests-php.md

```diff
@@ -323,15 +323,56 @@ then you should update the expected system files accordingly. To compare and upd
 
 * Find out the Travis build number by opening the Travis run for your pull request. The build number is typically a 5 or 6 digit number and has a leading hash character. For example when the build is `#45511`, then `45511` is the build number.
 * Execute this command and replace `{buildnumber}` with the actual build number. `./console development:sync-system-test-processed {buildnumber}`.
-  * To update the expected files directly append the option `--expected`. You then need to make sure before committing and pushing these changes that every change is actually expected
+  * To update the expected files directly append the option `--expected`. You then need to make sure before committing and pushing these changes that every change is actually expected.
   * Or if you only want to update some files or if you don't use a visual tool for git then you can execute the command without the expected option in which case the system files are updated in the `processed` directory. For example `tests/PHPUnit/System/processed` and `plugins/Goals/tests/System/processed`. If you are using PHPStorm you can then select both the processed and expected directory and then `right click -> Compare Directories`. This allows you to review every change of added, changed and removed files and let's you update each expected file individually. 
-* Then `git add` and `git commit` and `git push` the changes to trigger another build run 
-* If some tests are still failing you may need to repeat this process as sometimes you might forget to update some
+* Then `git add` and `git commit` and `git push` the changes to trigger another build run. 
+* If some tests are still failing you may need to repeat this process as sometimes you might forget to update some.
 
 ## Debugging tests
 
 As a software developer writing tests it can be useful to be able to set breakpoints and debug while running tests. If you use Phpstorm [read this answer](http://stackoverflow.com/a/14998884/3759928) to learn to configure Phpstorm with the PHPUnit from Composer.
 
+The following configuration can be useful also to debug from the command line:
+
+```ini
```

Comment:
@geekdenz I think this might be better suited for https://developer.matomo.org/guides/php and when needed we can link from this guide to the PHP guide.

Reply:
from https://developer.matomo.org/guides/debugging-core we can link to the PHP guide then as well.